### ISSUE TITLE
Print latest packet sequences in log or app.yaml

### ISSUE THREAD

**Issue by orkunkl** [confio/ts-relayer#146]
Helpful when debugging and tracking current status of relayer

**Comment by bartmacbartek** [comment]
There's `last-queried-heights.json` file at `$HOME/.ibc-setup` directory (by default):
```ts
type LastQueriedHeights = {
  packetHeightA: number,
  packetHeightB: number,
  ackHeightA: number,
  ackHeightB: number
}
```

Is it any helpful? If not, could you elaborate a bit more on what you would like to see in logs, please?

**Comment by orkunkl** [comment]
Not exactly, `LastQueriedHeights` is actually shown in the logs while the relayer is running, but packet height and ack height indicates the latest heights a packet sent and acknowledged.
Packets has a sequence identifier inside that is sequential and would be helpful to see this in the logs.

```
interface Packet {
  sequence: uint64
  timeoutHeight: Height
  timeoutTimestamp: uint64
  sourcePort: Identifier
  sourceChannel: Identifier
  destPort: Identifier
  destChannel: Identifier
  data: bytes
}
```

**Comment by ethanfrey** [comment]
I cannot add it as `info` level, as we print one line for all packets:

`Relay ${packets.length} packets from ${this.chain(source)} => ${this.otherChain(source)}`

We do show it in verbose mode where we show the data of each packet:

`Sending packet #${packet.sequence.toNumber()} from ${this.chainId}:${packet.sourceChannel}`, `presentPacketData(packet.data)`

This contains the channel and sequence along with the full packet application data (eg. ics20 source/receive/denom/amount).

You can just run with `-v` to get this info. Or is there some more info you need?

**Comment by orkunkl** [comment]
It has been some time since I created this request. I should have explained the issue better in the description. If packet height provided using verbose mode, all fine.
